views/extras.tmpl

```diff
@@ -275,18 +275,10 @@
                   {{end}}
               {{end}}
               <td class="addr-tx-time">
-                  {{if eq .Confirmations 0}}
-                      <span data-target="address.formattedTime">Unconfirmed</span>
-                  {{else}}
-                      {{.Time}}
```

Comment:
The address controller, and the formattedTime target, only acts on rows that are unconfirmed or were unconfirmed at the time of page load.  This seems error prone.